models/points.go

```diff
@@ -26,104 +27,158 @@ var (
 		'=': []byte(`\=`),
 	}
 
-	ErrPointMustHaveAField  = errors.New("point without fields is unsupported")
-	ErrInvalidNumber        = errors.New("invalid number")
-	ErrInvalidPoint         = errors.New("point is invalid")
-	ErrMaxKeyLengthExceeded = errors.New("max key length exceeded")
```

Comment:
Removed this error - we still report max key length exceeded, but the error is constructed at runtime. `ErrMaxKeyLengthExceeded` is still an exported error in the tsm1 package.